### DISCUSSION TITLE
Is this weird checkerbox thing supposted to be there?

### DISCUSSION THREAD

**Discussion by joman66** [ntsc-rs/ntsc-rs#188] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/dc8dd163-7419-40db-ba4d-1e0149dc0ccb)
This is a 300x view of a 480p SMPTE Color Bars. It literally looks like I can see through the colors like it's transparent or something.
I've used NTSC-rs over actual footage and I've seen this checkerbox effect between where these colors appear. How do I remove them? Is it a side-effect to the plugin's algorithm or something? Thanks!

**Comment by valadaptive** [comment, marked as answer]
This is an artifact that exists in real NTSC footage known as [dot crawl](https://en.wikipedia.org/wiki/Dot_crawl). There are a few settings that seem to affect its prevalence:
- The "Chroma demodulation filter" has the largest effect. Changing it to "Box" will greatly reduce the artifacts, and either of the "comb" options should eliminate them entirely at the expense of some vertical blurring.
- The "Ringing" setting boosts high-frequency artifacts like dot crawl, and disabling it will make those artifacts less present.
- Increasing the "Luma smear" setting will blur the image horizontally, but this also has the effect of smoothing out dot crawl.
